package-lock.json

```diff
@@ -7951,9 +7951,9 @@
 			}
 		},
 		"node_modules/micromatch": {
-			"version": "4.0.7",
-			"resolved": "https://registry.npmjs.org/micromatch/-/micromatch-4.0.7.tgz",
-			"integrity": "sha512-LPP/3KorzCwBxfeUuZmaR6bG2kdeHSbe0P2tY3FLRU4vYrjYz5hI4QZwV0njUx3jeuKe67YukQ1LSPZBKDqO/Q==",
+			"version": "4.0.8",
```

Comment:
we should be able to revert this 👀

Reply:
Yeah not sure how that got through 😅. I know I ran the npm audit when I was installing pacakges after the the TS update but I don't remember actually running npm audit fix so I don't really know what happened there.

Reply:
I'm happy to revert this, I'm not fully sure how. I think from what I'm googling I need to find the hash where it is the original and check out and commit again. Is that the correct track?

I was also looking at what is on main right now and I see it is also 4.0.8, will it cause any issues if I revert and then merge?